### ISSUE TITLE
Chords misplaced with whole rest note Measure

### ISSUE THREAD

**Issue by djae88** [opensheetmusicdisplay/opensheetmusicdisplay#791]
![chord1](https://user-images.githubusercontent.com/56290540/84700894-f3e04700-af8e-11ea-8ea5-30f8580d8322.png)
↑ Sibelius
![image](https://user-images.githubusercontent.com/56290540/84701415-e8415000-af8f-11ea-8979-6c4d57a45e8e.png)
↑ Finale

I tried to put the chords on the first and third beats in first measure
and first and second beats in second measure

But it didn't work
![chord2](https://user-images.githubusercontent.com/56290540/84701144-62250980-af8f-11ea-9392-d7f83237f1a0.png)
↑ OSMD

I saw this #599 comment
But I think it's a little different from this one,
so I'm writing a new one

I think the chord is connected to a note in the same measure
But it seems to cause a lot of trouble
and how about consider using the beat to locate the chords?

and when the measure with whole rest note and chords
instrumentalist will want to see the first chords on first beat
but right now, In OSMD it shows at the center of measure
![image](https://user-images.githubusercontent.com/56290540/84702283-51759300-af91-11ea-9aaf-99be25bb60db.png)
↑ OSMD (single staff)

Best regards

**Comment by bneumann** [comment]
Hi,

this is a little tricky. I made a measure like so in Musescore:
![image](https://user-images.githubusercontent.com/2605131/89122964-075a5980-d4cc-11ea-91f1-3fabeb5de39a.png)

I imported it again:
![image](https://user-images.githubusercontent.com/2605131/89122967-13461b80-d4cc-11ea-9825-d2fd86beb92b.png)

The problem lies in the XML Musescore produces:

```xml
      <harmony print-frame="no">
        <root>
          <root-step>D</root-step>
          </root>
        <kind>major</kind>
        </harmony>
      <harmony print-frame="no">
        <root>
          <root-step>C</root-step>
          </root>
        <kind>major</kind>
        </harmony>
```

It contains no information about the placement of the chord. So our aproach is to take the note/rest next to the xml node and assign the chord to that note/rest.
I stumbled over the same issue while writing lead sheets for my band. And I found a workaround to make a second voice with invisible quarter rests and assign the Chords to them.

Can you give me the XML you created with Sibelius? Maybe they provide more information.

**Comment by djae88** [comment]
I'm sorry I'm late, here's the sample xml
I lost the old one, so I made the same one.

![chord whole rest](https://user-images.githubusercontent.com/56290540/89871532-83525100-dbf2-11ea-9e5e-637f230c46a3.png)
[Chord whole rest.zip](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/files/5055521/Chord.whole.rest.zip)

![single staff](https://user-images.githubusercontent.com/56290540/89871576-95cc8a80-dbf2-11ea-8d73-b0be467e57f6.png)
[Chord test (single staff).zip](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/files/5055522/Chord.test.single.staff.zip)

in single staff xml
![sdfsfd](https://user-images.githubusercontent.com/56290540/89872830-7afb1580-dbf4-11ea-9657-c8722040f7fa.png)
I think this means duration of measure ( 256 x 4 )
![dfgdgrege](https://user-images.githubusercontent.com/56290540/89873076-d927f880-dbf4-11ea-8c38-9fc9105baaa0.png)
So in this xml duration 512 means half note 
and I think they're trying to put information about the placement of the chord like this
and It seems to work well betwenn Sibelius and the finale

I already found a way to solve this manually just like yours
but it's annoying to do this every time hhh....

I hope this information will help you solve this problem
Best wishes

**Comment by sschmidTU** [comment]
This was closed by accident, the above commit has nothing to do with this.

**Comment by TormodKv** [comment]
Any progress on this? 😅
Or does anyone have a workaround which doesn't involve changing the musicxml?

**Comment by sschmidTU** [comment]
Fixed in #1446.

<img width="201" alt="image" src="https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/assets/33069673/df2a8a4b-07ed-440f-bc3e-ec42ed977a96">
